### PR TITLE
Cog 1210 add section field in pycognaize sdk

### PR DESCRIPTION
**Pull request by hovhannes-zohrabyan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.4]
+
+### [1.4.0] - 2023-05-29
+- Add section field and section tag functionality
```
Comment (davidcognaize) [inline CHANGELOG.md:6]: Increment version number

File: pycognaize/document/field/section_field.py [hunk 2]
```diff
@@ -38,10 +39,23 @@ def __init__(self,
                          group_key=group_key, confidence=confidence,
                          group_name=group_name)
         self._field_id = field_id
+        self._value = value
+        self._ocr_value = ocr_value
+
+    @property
+    def start_tag(self):
```
Comment (davidcognaize) [inline pycognaize/document/field/section_field.py]: Why not start and end instead of start_tag end_tag?

**Review by davidcognaize — CHANGES_REQUESTED** [review]

File: pycognaize/document/field/section_field.py (hunk 2 above)
Comment (hovhannes-zohrabyan) [reply pycognaize/document/field/section_field.py]: Because if specifies what the start is (Coordinates, value, tag or etc)

Comment (hovhannes-zohrabyan) [reply pycognaize/document/field/section_field.py:46]: @davidcognaize

**Review by davidcognaize — APPROVED** [review]

**Pull request merged**